examples/ppo/env.py

```diff
@@ -0,0 +1,57 @@
+import collections
+import gym
+import numpy as onp
+
+from seed_rl_atari_preprocessing import AtariPreprocessing
+
+class ClipRewardEnv(gym.RewardWrapper):
+  """This class is adatpted from OpenAI baselines
```

Comment:
Hey @wrzadkow Thank you so much - PPO is not an easy one to port to Flax, so this awesome. I've noticed you are using different Python docstrings formats for multiple lines. Maybe, to automate the process, you could run the code through [`pydocstyle`](http://www.pydocstyle.org) and/or use this handy [Python Docstring Generator](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) extension in VSCode. The latter uses the Google-style docstring format by default and also takes care of arguments/parameters, errors and returns. Also, keep an eye on typos :) ("adatpted") I'm sure you'll polish it all up before the final version. Thanks for the hard work!

Reply:
Now this commit https://github.com/google/flax/pull/470/commits/f53c1df003608da37e683891d1e1b5503fe3724a includes runnig `pydocstyle` over the entire example. I also ran a spellchecker. Thanks for the feedback, hinting towards `pydocstyle` and all the kind words!

Reply:
@wrzadkow Thank you so much! When ML/RL code is readable, it's a delight to study and learn from 👏